frontend/cypress/e2e/FormStaffPage.cy.ts

```diff
@@ -395,6 +397,283 @@ describe("Staff Form", () => {
             });
           });
         });
+
+        /*
+        Helper function to help organize the test steps in a way that resembles the way you would
```

Comment:
Why this approach instead of using `describe` and `it`? Is there any issues with it?